SendUserDataProcessingEmailActivity/index.ts

```diff
@@ -0,0 +1,72 @@
+﻿import * as NodeMailer from "nodemailer";
+
+import { getRequiredStringEnv } from "io-functions-commons/dist/src/utils/env";
+
+import { DocumentClient as DocumentDBClient } from "documentdb";
+import {
+  PROFILE_COLLECTION_NAME,
+  ProfileModel
+} from "io-functions-commons/dist/src/models/profile";
+import * as documentDbUtils from "io-functions-commons/dist/src/utils/documentdb";
+import { MailUpTransport } from "io-functions-commons/dist/src/utils/mailup";
+import { getSendUserDataProcessingEmailActivityHandler } from "./handler";
+
+const cosmosDbUri = getRequiredStringEnv("COSMOSDB_URI");
+const cosmosDbKey = getRequiredStringEnv("COSMOSDB_KEY");
+const cosmosDbName = getRequiredStringEnv("COSMOSDB_NAME");
+
+const documentDbDatabaseUrl = documentDbUtils.getDatabaseUri(cosmosDbName);
+
+const profilesCollectionUrl = documentDbUtils.getCollectionUri(
+  documentDbDatabaseUrl,
+  PROFILE_COLLECTION_NAME
+);
+
+const documentClient = new DocumentDBClient(cosmosDbUri, {
+  masterKey: cosmosDbKey
+});
+
+const profileModel = new ProfileModel(documentClient, profilesCollectionUrl);
+
+// Whether we're in a production environment
+const isProduction = process.env.NODE_ENV === "production";
+
+// Mailup
+const mailupUsername = getRequiredStringEnv("MAILUP_USERNAME");
+const mailupSecret = getRequiredStringEnv("MAILUP_SECRET");
+
+// Email data
+const mailFrom = getRequiredStringEnv("DPO_EMAIL_FROM_ADDRESS");
```

Comment:
sorry I've changed my mind about this and it's ok to use the global mail_from setting (it's no-reply@io.italia.it)
:))

```suggestion
const mailFrom = getRequiredStringEnv("MAIL_FROM");
```

Reply:
Ok let's refactoring!